automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -38,6 +38,17 @@ main() {
     source automation/check-patch.setup.sh
     cd ${TMP_PROJECT_PATH}
 
+    kubectl apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: cluster-network-addons
+  labels:
+    control-plane: mac-controller-manager
```

Comment:
I think adding/removing the `control-plane` label should not be part of this PR,